doc/sphinx/language/gallina-extensions.rst

```diff
@@ -83,9 +83,13 @@ To build an object of type :token:`ident`, one should provide the constructor
 
 .. _record-named-fields-grammar:
 
+  .. insertgram fields_def field_def
+
   .. productionlist::
-    record_term : {| [`field_def` ; … ; `field_def`] |}
-    field_def : `ident` [`binders`] := `term`
+     fields_def : `field_def` ; `fields_def`
+                : `field_def`
+                : `empty`
+     field_def : `qualid` `binders_opt` := `term`
```

Comment:
This grammar is almost pointless in this context: it is not described at all in the surrounding text. Can you just move the definition of `fields_def` and `field_def` in the main grammar chunk in the Gallina chapter? The same is not true for the projection grammar chunk below, which is described in the surrounding text.